### PR TITLE
Fix/linting api

### PR DESCRIPTION
**Pull request by nikrb**
fixed api linting

### CODE REVIEW

**Review by g1st — DISMISSED** [review]
Looks good, thank you.

File: api/models/works.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
 const mongoose = require('mongoose');
 
+function toLowerCase(val) {
+  const v = typeof val !== 'string' ? val : '';
+  return v.toLowerCase();
+}
+
```
Comment (zurda) [inline api/models/works.js:7]: I think this amended functions is a bit different from the original. In the original, if the type of value isn't a string -> we assign `val` an empty string. I suggest writing it like so: 

```js
function toLowerCase(val) {
  const v = typeof val === 'string' ? val : '';
  return v.toLowerCase();
}
```

**Review by zurda — CHANGES_REQUESTED** [review]
Added a comment about the `toLowerCase` function, but other than that LGTM 👍

File: api/models/works.js (hunk 1 above)
Comment (g1st) [reply api/models/works.js:7]: Good catch!

**Review by zurda — APPROVED** [review]

**Review by g1st — APPROVED** [review]

**Pull request merged**